### DISCUSSION TITLE
关于自动引用的问题（About automatically quoted）

### DISCUSSION THREAD

**Discussion by hanzhe910** [windingwind/zotero-better-notes#655] [Q&A] [answered]
求助，每次从论文中复制粘贴到better note中的时候都会自动引用并且有引注，请问应该如何取消这个功能呢
Please help, every time I copy and paste something from a paper into the better note it is automatically quoted and has a citation, how do I cancel this function?

**Comment by windingwind** [comment, marked as answer]
<img width="513" alt="image" src="https://github.com/windingwind/zotero-better-notes/assets/33902321/e4e50fa3-4a18-4015-9f6a-6aac259739ce">
